corporate/lib/remote_billing_util.py

```diff
@@ -77,7 +85,10 @@ def get_remote_server_from_session(
     request: HttpRequest,
     server_uuid: str,
 ) -> RemoteZulipServer:
-    identity_dict = get_identity_dict_from_session(request, None, server_uuid)
+    identity_dict: Optional[LegacyServerIdentityDict] = get_identity_dict_from_session(
+        request, None, server_uuid
```

Comment:
Can we switch this to using mandatory kwargs? A positional `None` parameter is always annoying to read.